html/modules/custom/ncms_ui/src/Entity/Content/ContentBase.php

```diff
@@ -416,4 +417,34 @@ public function getCommonTaxonomiesService() {
     return \Drupal::service('ncms_tags.common_taxonomies');
   }
 
+  /**
+   * Builds the render array for displaying the current results as a table.
+   */
+  public function getTags() {
+    /** @var \Drupal\ncms_tags\CommonTaxonomyService $common_taxonomies */
+    $common_taxonomies = \Drupal::service('ncms_tags.common_taxonomies');
```

Comment:
This should use the `getCommonTaxonomiesService()` method from just above this method.